src/gsi/gsi_obOperTypeManager.F90

```diff
@@ -363,7 +367,6 @@ function dtype2index_(dtype) result(index_)
   case("aero"   ,"[aerooper]"   ); index_= iobOper_aero
     case("aod"      ); index_= iobOper_aero
     case("modis_aod"); index_= iobOper_aero
-    case("viirs_aod"); index_= iobOper_aero
```

Comment:
Why is this line removed?

Reply:
Updated this section to match the Develop branch.